src/frontend/classic/App.svelte

```diff
@@ -96,70 +82,14 @@
     $: currentContent = selectedItem ? "viewer" : "home";
 </script>
 
-<Theme persist bind:theme>
-    <Header
-        id="Header"
-        expandedByDefault={false}
-        persistentHamburgerMenu={true}
-        company="HakuNeko"
-        platformName="Media & Anime - Downloader"
-        bind:isSideNavOpen
-    >
-        <div slot="skip-to-content">
-            <SkipToContent />
-        </div>
-        <HeaderUtilities>
-            <HeaderGlobalAction
-                on:click={() => win.minimize()}
-                aria-label="Minimize"
-                icon={MinimizeIcon}
-            />
-            <HeaderGlobalAction
-                on:click={() => (winMaximized ? win.restore() : win.maximize())}
-                aria-label="Maximize"
-                icon={winMaximized ? RestoreIcon : MaximizeIcon}
-            />
-            <HeaderGlobalAction
-                on:click={() => win.close()}
-                aria-label="Close"
-                icon={CloseIcon}
-            />
-            <HeaderAction bind:isOpen>
-                <HeaderPanelLinks>
-                    <HeaderPanelDivider>Interface</HeaderPanelDivider>
-                    <HeaderPanelLink>
-                        <Toggle
-                            size="sm"
-                            labelText="Show content pannel :"
-                            labelA="Download Only"
-                            labelB="Content"
-                            toggled={uimode === "ui-mode-content"}
-                            on:toggle={changeUIMode}
-                        />
-                    </HeaderPanelLink>
-                    <HeaderPanelDivider>Themes</HeaderPanelDivider>
-                    {#each themes as item}
-                        <HeaderPanelLink on:click={() => (theme = item.id)}
-                            >{item.label}</HeaderPanelLink
-                        >
-                    {/each}
-                </HeaderPanelLinks>
-            </HeaderAction>
-        </HeaderUtilities>
-    </Header>
-
-    <SideNav bind:isOpen={isSideNavOpen}>
-        <SideNavItems>
-            <SideNavLink text="Link 1" />
-            <SideNavLink text="Link 2" />
-            <SideNavLink text="Link 3" />
-            <SideNavMenu text="Menu">
-                <SideNavMenuItem href="/" text="Link 1" />
-                <SideNavMenuItem href="/" text="Link 2" />
-                <SideNavMenuItem href="/" text="Link 3" />
-            </SideNavMenu>
-        </SideNavItems>
-    </SideNav>
+<Theme bind:theme>
+    <AppBar
+        {isSideNavOpen}
+        {isOpen}
+        {winMaximized}
+        {changeUIMode}
+        {changeTheme}
+    />
 
     <Content id="hakunekoapp">
         <MediaSelect on:select={(e) => (selectedMedia = e.detail)} />
```

Comment:
May remove redundant brackets `e => ...` instead of `(e) => ...`.
May use more expressive variable names, e.g. `evt` or `event` instead of `e`

Reply:
I change `e` to `evt`, but the brackets can't be removed with the current linter config